src/remote/os/win32/xnet.cpp

```diff
@@ -307,7 +307,8 @@ rem_port* XNET_analyze(ClntAuthBlock* cBlock,
 		REMOTE_PROTOCOL(PROTOCOL_VERSION15, ptype_batch_send, 6),
 		REMOTE_PROTOCOL(PROTOCOL_VERSION16, ptype_batch_send, 7),
 		REMOTE_PROTOCOL(PROTOCOL_VERSION17, ptype_batch_send, 8),
-		REMOTE_PROTOCOL(PROTOCOL_VERSION18, ptype_batch_send, 9)
+		REMOTE_PROTOCOL(PROTOCOL_VERSION18, ptype_batch_send, 9),
+		REMOTE_PROTOCOL(PROTOCOL_VERSION19, ptype_lazy_send, 10)
```

Comment:
Shouldn't be it `ptype_batch_send` as before ?

Reply:
Should be, thanks.